packages/core/types/ts3.6/runSaga.test.ts

```diff
@@ -7,7 +7,8 @@ declare const effect: StrictEffect
 declare const iterator: Iterator<any>
 
 function testRunSaga() {
-  const task0: Task = runSaga<{ foo: string }, { baz: boolean }, () => SagaIterator>(
+  // $ExpectType Task<void>
+  const task0 = runSaga<{ foo: string }, { baz: boolean }, Saga<undefined[], void>>(
```

Comment:
It feels like you might have wanted to just provide no arguments:
```suggestion
  const task0 = runSaga<{ foo: string }, { baz: boolean }, Saga<[], void>>(
```

Reply:
Tried that and got this error:

```
ERROR: 11:60  expect  Compile error in typescript@3.7 but not in typescript@3.8.
Fix with a comment '// Minimum TypeScript Version: 3.8' just under the header.
Type 'Saga<[], void>' does not satisfy the constraint 'Saga<any[], any>'.
  Type 'any[]' is not assignable to type '[]'.
    Types of property 'length' are incompatible.
      Type 'number' is not assignable to type '0'.
```

So left as-is.